frontend/src/pages/Landing/LandingPage.tsx

```diff
@@ -208,7 +225,7 @@ function LandingPage() {
                 <img
                   src="img/118172286-e3fffb80-b3c7-11eb-9553-127462881af3.png"
                   className={styles.PrioritizeHealthEquityImg}
-                  alt="Three women embracing a small baby in between them"
```

Comment:
Same here

Reply:
It's a judgement call, but from what I've researched and the guidance given on that A11y spreadsheet, alt text for images should be a blank string `alt=""` for any decorative images, making the screen reader intentionally skip the element rather than try and make sense of it. Although the photos provide an inclusive atmosphere for sighted users, someone using a screen reader might be confused as to why the photograph is there, and if it serves a functional or informational purpose (which I decided they don't).

Reply:
Cool, makes sense to me